### PR TITLE
Add Adobe Source Han Sans font for CJK

### PR DESCRIPTION
**Pull request by mmoskal**
Waiting on OSS approval.

Should be used with https://github.com/microsoft/pxt/pull/5584 otherwise it will blow up the binary

Test build: https://arcade.makecode.com/app/bb61882806f3c6cef0936bbe9f54b7217068eb6a-65b7c7e45d

### CODE REVIEW

File: libs/game/prompt.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ namespace game {
 
 
     //% whenUsed=true
-    const font = image.font8;
+    const font = image.font8; // FONT8-TODO
```
Comment (mmoskal) [inline libs/game/prompt.ts]: @riknoll if you want to have a look

**Comment by pelikhan** [comment]
@abchatra @mmoskal bump, any progress?

**Comment by mmoskal** [comment]
waiting on legal

**Comment by abchatra** [comment]
Review seemed to be completed and approved https://ossmsft.visualstudio.com/_oss?requestId=1704535&_a=progress

**Comment by riknoll** [comment]
Note the legal guidance in that link has some requirements around the third party notices

**Comment by pelikhan** [comment]
@mmoskal

**Comment by mmoskal** [comment]
@pelikhan should be good now

**Pull request merged**